### ISSUE TITLE
Ansible provider will not copy a vault file whose name starts with a dot

### ISSUE THREAD

**Issue by marcoamorales** [neillturner/kitchen-ansible#104]
If I have the following: `.kitchen.yml`:

```
...
  ansible_vault_password_file: .vault_pass.txt
...
```

I will get the following error message:
```
       Going to invoke ansible-playbook with: ANSIBLE_ROLES_PATH=/tmp/kitchen/roles sudo -E ansible-playbook --inventory-file=/tmp/kitchen/hosts -c local -M /tmp/kitchen/modules -v   --vault-password-file=/tmp/kitchen/.vault_pass.txt    /tmp/kitchen/app.yaml
       Traceback (most recent call last):
         File "/usr/bin/ansible-playbook", line 324, in <module>
           sys.exit(main(sys.argv[1:]))
         File "/usr/bin/ansible-playbook", line 142, in main
           vault_pass = utils.read_vault_file(options.vault_password_file)
         File "/usr/lib/pymodules/python2.7/ansible/utils/__init__.py", line 176, in read_vault_file
           if is_executable(this_path):
         File "/usr/lib/pymodules/python2.7/ansible/utils/__init__.py", line 291, in is_executable
           return (stat.S_IXUSR & os.stat(path)[stat.ST_MODE]
       OSError: [Errno 2] No such file or directory: '/tmp/kitchen/.vault_pass.txt'
>>>>>> Converge failed on instance <default-ubuntu-1404>.
>>>>>> Please see .kitchen/logs/default-ubuntu-1404.log for more details
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: SSH exited (1) for command: [ANSIBLE_ROLES_PATH=/tmp/kitchen/roles sudo -E ansible-playbook --inventory-file=/tmp/kitchen/hosts -c local -M /tmp/kitchen/modules -v   --vault-password-file=/tmp/kitchen/.vault_pass.txt    /tmp/kitchen/app.yaml]
>>>>>> ----------------------
zlib(finalizer): the stream was freed prematurely.
```

If I just remove the dot in the filename, it will work :confused:.

I'm running:

- kitchen-ansible (0.0.31)
- kitchen-vagrant (0.19.0)
- test-kitchen (1.4.2)
- Mac OS X 10.11.1
- Ruby 2.1.6 (through chefdk)

**Comment by neillturner** [comment]
a couple of things. 
1, run with your kitchen command with -l debug to get more messages. 
2. after running log on to the server and see if your vault file is in /tmp/kitchen
3. if it is try changing adding quotes around the vault file in the command by changing line in 
ansible_playbook.rb around line 576. 
config[:ansible_vault_password_file] ? "--vault-password-file=#{File.join(config[:root_path], File.basename(config[:ansible_vault_password_file]))}" : nil
      end
to 
config[:ansible_vault_password_file] ? "--vault-password-file='#{File.join(config[:root_path], File.basename(config[:ansible_vault_password_file]))}'" : nil
      end
4. if it doesn't copy then put in the messages and see why it doesn't copy.

**Comment by lobsterdore** [comment]
I just ran into the same issue, the vault file gets uploaded but doesn't get copied to /tmp/kitchen if it starts with a dot.

**Comment by neillturner** [comment]
fixed in version 0.0.32

**Comment by lobsterdore** [comment]
Awesome work, will test later on today.
